app/src/main/res/values-land/dimens.xml

```diff
@@ -42,6 +42,9 @@
   <dimen name="profile_chooser_profile_view_margin_start">16dp</dimen>
   <dimen name="profile_chooser_profile_view_margin_end">16dp</dimen>
 
+  <!-- profile-list-profile-view -->
```

Comment:
Rename to 
`<!-- Profile List Profile View -->`

Ditto everywhere else.

Reply:
Okay, will make the required changes.